src/utils.ts

```diff
@@ -24,6 +26,23 @@ export function getCheckRunPayloadWithScanResult(trivyStatus: number, dockleStat
     return checkRunPayload;
 }
 
+export function getScanReportPath(): string {
+  const scanReportPath = `${fileHelper.getContainerScanDirectory()}/scanReport.json`;
+  return scanReportPath;
+}
+
+export function createScanReport() {
+  const scanReportPath = getScanReportPath();
+  const trivyOutput = trivyHelper.getTrivyFilteredOutput();
+  const dockleOutput = dockleHelper.getDockleFilteredOutput();
+  const scanReportObject = {
+    "commonVulnerabilities": trivyOutput,
+    "bestPracticeVulnerabilities": dockleOutput
+  };
+  const output = `{\n\t"commonVulnerabilities": ${trivyOutput},\n\t"bestPracticeVulnerabilities": ${dockleOutput}\n}`;
+  fs.writeFileSync(scanReportPath, JSON.stringify(scanReportObject, null, 2));
+}
```

Comment:
This logic can be moved to main.ts

Reply:
I thought there's already too much content in main, so it's better to keep it in utils.